packages/devtools_app/lib/src/flutter/scaffold.dart

```diff
@@ -80,11 +82,16 @@ class DevToolsScaffoldState extends State<DevToolsScaffold>
   /// coordinate their animation when the tab selection changes.
   TabController _tabController;
 
+  Screen _currentScreen;
+  final StreamController<Screen> _screenController =
```

Comment:
instead of tracking _currentScreen and _screenController, 
consider a single
`ValueNotifier _currentScreen;`
https://api.flutter.dev/flutter/foundation/ValueNotifier-class.html
In general, we've found ValueNotifier is usually a bit cleaner for Flutter code than Stream as it has a better notion of the current value and has less functionality that is irrelevant for UI code.